benchmark/single-source/SetTests.swift

```diff
@@ -12,69 +12,115 @@
 
 import TestsUtils
 
+let size = 400
+let overlap = 100
+
+let setAB = Set(0 ..< size)
```

Comment:
I was drawing number lines like the dummy that I am… a little bit of spelling out would help me grok the correctness of this much sooner. What do you think?
````swift
let setAB = Set(0 ..< size)                            // 0...399
let setCD = Set(size ..< 2 * size)                     // 400...799
let setBC = Set(size - overlap ..< 2 * size - overlap) // 300...699
let setB = Set(size - overlap ..< size)                // 300...399
````